src/cascadia/TerminalApp/App.xaml

```diff
@@ -45,7 +45,7 @@
                                      Color="{ThemeResource SystemErrorTextColor}" />
 
                     <!--  Suppress top padding  -->
-                    <Thickness x:Key="TabViewHeaderPadding">9,0,5,0</Thickness>
+                    <Thickness x:Key="TabViewHeaderPadding">0,0,0,0</Thickness>

```

Comment:
removing the right padding - worth making sure about the sice when the tab bar is full-width

Reply:
I _cannot_ figure out what the right padding was for. It looks the same here as in 1.18.1291 (and this PR isn't in that build)